app/models/communication_log.rb

```diff
@@ -9,10 +9,10 @@ class CommunicationLog < ApplicationRecord
 
   scope :needs_follow_up, ->(boolean=nil){ where(needs_follow_up: boolean || true) }
 
-  def self.log_submission_email(email_object, delivery_status, submission, delivery_method=nil, current_user=nil)
+  def self.log_email(email_object, delivery_status, person, delivery_method=nil, current_user=nil)
```

Comment:
Nice change and reuse!

Reply:
we were just talking about this!

Reply:
I noticed that method and figured why not.